src/test/smart-components/portfolio/share-portfolio-modal.test.js

```diff
@@ -83,32 +78,32 @@ describe('<SharePortfolioModal/>', () => {
     jest.useFakeTimers();
     expect.assertions(1);
     const store = mockStore(initialState);
+
     /**
      * download data endpoints
      */
     mockApi
       .onGet(`${CATALOG_API_BASE}/portfolios/2/share_info/`)
       .replyOnce(200, {
-        results: []
+        data: []
       });
-    mockApi.onGet(`${CATALOG_API_BASE}/groups/`).replyOnce(200, {
-      results: [{ id: '123', name: 'Group 123' }]
+    mockApi.onGet(`${RBAC_API_BASE}/groups/`).replyOnce(200, {
+      data: [{ uuid: '123', name: 'Group 123' }]
     });
+
     /**
      * submit data endpoints
      */
     mockApi
-      .onPost(`${CATALOG_API_BASE}/portfolios/2/share/`)
+      .onPost(`${CATALOG_API_BASE}/portfolios/2/share`)
```

Comment:
Curious why this API call changed, I thought we had to include the trailing `/` otherwise there were issues?